src/test/java/com/thealgorithms/backtracking/CombinationTest.java

```diff
@@ -1,6 +1,7 @@
 package com.thealgorithms.backtracking;
 
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
 
 import java.util.List;
```

Comment:
Regarding the failing build, try to:

```suggestion
import java.util.Arrays;
import java.util.List;
```